### PR TITLE
Create LICENSE.md

### PR DESCRIPTION
**Pull request by andreakarasho**
First of all I'd like to know what do you think about. From other guys too.
To pass from GPL3 to BSD i need the approvation of all contributors that wrote important code for ClassicUO.
I'll tag all of you guys just to speed up things.

In order:
@zolter @roxya @3HMonkey @Stallind @plugtopus @markdwags @brndd @jkachhad @DarkLotus @rdegelo @binff04 @Hezkore @Syrupz-UO @dissident76 @JakubLinhart @TannerBist @paezao @aclark13 @SaschaKP @mngraves @c0mput3rxz 
@Mentos- @JulianUO @tonylittle306 @canterberry @luuutz @simarsenault @jinh86 @MayDF 

### Why?
GPL3 puts serious limits to the software around ClassicUO and to ClassicUO itself.

### How to answer
Write a new post with YES or NO.

### What's happen if i say 'NO' ?
I have to delete the code you wrote if it has an important impact or i'll just keep GPLv3. Minor mods are not contempled.

### BSD 4 License text
```
Copyright (c) 2020, andreakarasho
All rights reserved.

Redistribution and use in source and binary forms, with or without
modification, are permitted provided that the following conditions are met:
1. Redistributions of source code must retain the above copyright
   notice, this list of conditions and the following disclaimer.
2. Redistributions in binary form must reproduce the above copyright
   notice, this list of conditions and the following disclaimer in the
   documentation and/or other materials provided with the distribution.
3. All advertising materials mentioning features or use of this software
   must display the following acknowledgement:
   This product includes software developed by andreakarasho - https://github.com/andreakarasho
4. Neither the name of the copyright holder nor the
   names of its contributors may be used to endorse or promote products
   derived from this software without specific prior written permission.

THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ''AS IS'' AND ANY
EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER BE LIABLE FOR ANY
DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
(INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```

### CODE REVIEW

**Comment by SaschaKP** [comment]
YES

**Comment by roxya** [comment]
Yes

**Comment by Stallind** [comment]
Yes

**Comment by JulianUO** [comment]
Yes

**Comment by binff04** [comment]
Yes

**Comment by Hezkore** [comment]
Yes

**Comment by simarsenault** [comment]
YES

**Comment by canterberry** [comment]
YES

**Comment by paezao** [comment]
YES

**Comment by UOForever** [comment]
Yes

**Comment by Ultima-Tony** [comment]
Yes

**Comment by zolter** [comment]
YES

**Comment by brndd** [comment]
yes

**Comment by AsYlum-** [comment]
YES

**Comment by jaedan** [comment]
Yes

**Comment by Belialzao** [comment]
YES

**Comment by markdwags** [comment]
Yes

**Comment by dissident76** [comment]
Yes

-------- Original message --------From: Andrea Angelini <notifications@github.com> Date: 13/09/2020  03:22  (GMT+09:00) To: andreakarasho/ClassicUO <ClassicUO@noreply.github.com> Cc: dissident <dissident76@gmail.com>, Mention <mention@noreply.github.com> Subject: [andreakarasho/ClassicUO] Create LICENSE.md (#1214) First of all I'd like to know what do you think about. From other guys too.
To pass from GPL3 to BSD i need the approvation of all contributors that wrote important code for ClassicUO.
I'll tag all of you guys just to speed up things.
In order:
@zolter @roxya @3HMonkey @Stallind @plugtopus @markdwags @brndd @jkachhad @DarkLotus @rdegelo @binff04 @Hezkore @Syrupz-UO @dissident76 @JakubLinhart @TannerBist @paezao @aclark13 @SaschaKP @mngraves @c0mput3rxz
@Mentos- @JulianUO @tonylittle306 @canterberry @luuutz @simarsenault @jinh86 @MayDF
Why?
GPL3 puts serious limits to the software around ClassicUO and to ClassicUO itself.
How to answer
Write a new post with YES or NO.
What's happen if i say 'NO' ?
I have to delete the code you wrote if it has a important impact. Minor mods are not contempled or just keep GPLv3.



You can view, comment on, or merge this pull request online at:
  https://github.com/andreakarasho/ClassicUO/pull/1214

Commit Summary

  Create LICENSE.md


File Changes

  
    A
    LICENSE.md
    (27)
  


Patch Links:

  https://github.com/andreakarasho/ClassicUO/pull/1214.patch
  https://github.com/andreakarasho/ClassicUO/pull/1214.diff

**Comment by mngraves** [comment]
Yes

**Comment by 3HMonkey** [comment]
Yes

**Comment by Alias-Alan** [comment]
after review, hearing both sides of the argument. I think the project is better off with the current GPLv3 license. 

Further discussion and reasoning provided,
I vote yes to this license change.
@andreakarasho

**Comment by deccer** [comment]
To which of the `BSD` licenses are you referring to? There are at least 3 of them, all different.

**Comment by TannerBist** [comment]
Yes

**Comment by Syrupz-UO** [comment]
> To which of the `BSD` licenses are you referring to? There are at least 3 of them, all different.

I took this as BSD-ish (Clause 4)  which is why jadens running around commenting about licenses to the assistant owners. Clause 3 of BSD which is what makes it incompatible with the GPL unless they change licenses. If incorrect, please further explain.

**Comment by popoverde** [comment]
YES

**Comment by Mentos-** [comment]
Yes.

**Comment by MayDF** [comment]
Yes.

**Comment by aclark13** [comment]
YES

**Comment by JakubLinhart** [comment]
Yes

**Comment by JakubLinhart** [comment]
Please, could you elaborate on serious limits of GPL3?

**Comment by andreakarasho** [comment]
> Please, could you elaborate on serious limits of GPL3?

Mainly all code around the application under GPL3 must be licensed with GPL3 too. So all CUO fork and plugin must be open source.

Who loses here it's me only :)

**Comment by JakubLinhart** [comment]
So it is not possible to keep custom CUO modification in secret with GPL3?

**Comment by andreakarasho** [comment]
> So it is not possible to keep custom CUO modification in secret with GPL3?

Yes, for your personal use.
No if you distribuite it.

**Comment by gaechti** [comment]
> Who loses here it's me only :)

I beg to differ on that, potentialy any end user loses. If I understand this right (keeping custom custom cuo a secret). Just hypotecally speaking: if someone would distribute a backdoor implemented with cuo. while you would be protected by BSD, the distributing party would be too and could not be forced to reveal the source making it possible hard or impossible to proof.

however I totaly get the benefits too. E.g. Keith has put a lot of work in his custom version and may want to protect it. that can be rather hard opposed to e.g. server side scripts

should I got that wrong, I apology and I will edit it

**Comment by jkachhad** [comment]
YES

**Comment by jinh86** [comment]
YES

**Comment by DarkLotus** [comment]
YES

**Comment by c0mput3rxz** [comment]
yes

**Comment by brndd** [comment]
There's also some headaches with regards to using libraries with GPLed code, though to my understanding there should be no problem linking a GPLed project against a non-GPLed library; the problems mostly arise when linking non-GPLed projects against GPLed libraries.

Personally I prefer GPL purely because it means nobody in the future can fork ClassicUO and keep it closed source, but if the project leaders want to switch then that's up to them.

**Comment by andreakarasho** [comment]
We are psychologically ready to do this step i guess

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+1. Redistributions of source code must retain the above copyright
+   notice, this list of conditions and the following disclaimer.
+2. Redistributions in binary form must reproduce the above copyright
+   notice, this list of conditions and the following disclaimer in the
+   documentation and/or other materials provided with the distribution.
+3. All advertising materials mentioning features or use of this software
+   must display the following acknowledgement:
+   This product includes software developed by the copyright holder.
```
Comment (canterberry) [inline LICENSE.md]: This line should be:

```
This product includes software developed by andreakarasho.
```

Section 3 indicates that this acknowledgment must appear verbatim. In its current form, advertising materials would have to include literal text "This product includes software developed by the copyright holder." which carries no meaning in that context.

**Review by canterberry — COMMENTED** [review]
Because this changeset is quite large and difficult to manually review in the GitHub UI. I'd like to provide assurance to other reviewers that in comparing the diff of this PR against the target branch (`dev`), I can confirm that only licensing changes are included in this changeset and there do not appear to be any additional changes slipped in.

Methodology:

```
# Clone the `relicensing` branch of this repo locally
$ git clone -b relicensing git@github.com:andreakarasho/ClassicUO ClassicUO

# Switch to the clone's working directory
$ cd ClassicUO

# Fetch other branches from this remote (including dev)
$ git fetch --all

# Locally rebase against the latest dev branch
$ git rebase origin/dev

# Review only lines that would be *added* as a result of merging this branch into dev
$ git diff origin/dev..HEAD | grep -E '^\+[^+]' > 
```

**Comment by andreakarasho** [comment]
FYI: updated LICENSE text. I just fixed some minor and merged dev -> relicensing branch

**Pull request merged**